packages/portfolio-api/src/constants.js

```diff
@@ -15,6 +15,22 @@ export const YieldProtocol = /** @type {const} */ ({
 });
 harden(YieldProtocol);
 
+/**
+ * Operation types for wallet gas estimation. There are three types of wallet
+ * operations on a target EVM chain that incur different gas costs.
+ * Supply: user is supplying assets to a yield protocol on the target chain.
+ * Withdraw: user is withdrawing assets from a yield protocol on the target chain.
+ * DepositForBurn: user is moving assets from EVM chain via CCTP.
+ *
+ * @enum {(typeof WalletOperationType)[keyof typeof WalletOperationType]}
+ */
+export const WalletOperationType = /** @type {const} */ ({
+  Supply: 'supply',
+  Withdraw: 'withdraw',
+  DepositForBurn: 'depositforburn',
+});
+harden(WalletOperationType);
```

Comment:
"Wallet" is too overloaded in agoric-sdk to make a good unqualified name here.
```suggestion
 * @enum {(typeof EvmWalletOperationType)[keyof typeof EvmWalletOperationType]}
 */
export const EvmWalletOperationType = /** @type {const} */ ({
  Supply: 'supply',
  Withdraw: 'withdraw',
  DepositForBurn: 'depositforburn',
});
harden(EvmWalletOperationType);
```